docs/reference/ingest/apis/enrich/put-enrich-policy.asciidoc

```diff
@@ -144,3 +150,200 @@ to documents in the enrich index.
 Fields appended to incoming documents
 from matching documents in the enrich index.
 --
+
+[[put-enrich-policy-api-example]]
+==== {api-examples-title}
+
+[[put-enrich-policy-geo-match-ex]]
+===== Geo match policy type
+
+You can use the `geo_match` enrich policy type
+to enrich incoming documents
+based on matching geoshapes.
+For example,
+you can add postal codes
+to incoming documents
+based on a set of coordinates.
+
+To see how the geo match policy type works,
```

Comment:
I think this should also be a literal `geo_match`

Reply:
Fixed with 7ba28d77cf2bc0b37952c0ff3196b18064af8dc8:

https://github.com/elastic/elasticsearch/pull/47745/commits/7ba28d77cf2bc0b37952c0ff3196b18064af8dc8#diff-f1a2912a5d38f9cbe80f406513945ddbR168